src/main/java/com/rultor/agents/github/qtn/QnIfPull.java

```diff
@@ -77,16 +77,43 @@ public Req understand(final Comment.Smart comment,
         final URI home) throws IOException {
         final Issue.Smart issue = new Issue.Smart(comment.issue());
         final Req req;
-        if (issue.isPull()) {
-            req = this.origin.understand(comment, home);
-        } else {
+        if (QnIfPull.isNotPull(issue)) {
             new Answer(comment).post(
                 false,
                 QnIfPull.PHRASES.getString("QnIfPull.not-pull-request")
             );
-            req = Req.EMPTY;
+            req = Req.DONE;
```

Comment:
@dskalenko Ok but why does the new logic require this kind of change? Why adjust anything to the logic of handling this command being issued in a non-pull request while fixing the logic pertaining to actual pull requests?